docs/configuration/node-config.md

```diff
@@ -123,17 +123,7 @@ This section may contain one configuration subsection per available metastore im
 
 ### File-backed metastore configuration
 
-| Property | Description | Default value |
-| --- | --- | --- |
-| `polling_interval` | Time interval between successive polling attempts to detect metastore changes. | `30s` |
-
-Example of a metastore configuration for a file-backed implementation in YAML format:
-
-```yaml
-metastore:
-  file:
-    polling_interval: 1m
-```
+File-backed metastore doesn't have any node level configuration. You can configure the poll interval [at the index level](./metastore-config.md#polling-configuration).
```

Comment:
Can you quickly add the feature instead? There's already a `FileMetastoreConfig` in `metastore_config.rs`, and most of the boilerplate code is already set up. I suspect it won't take much work.

Reply:
Right, I was wondering if maybe it was implemented at some point and removed because of some perverse side effect.

Reply:
I'm merging this now to fix the docs and I'll add the feature request to my backlog. If nobody complained about this so far, the feature is likely not urgent.